glean-core/csharp/Glean/Glean.csproj

```diff
@@ -127,4 +127,12 @@
       <_Parameter1>GleanTests</_Parameter1>
     </AssemblyAttribute>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Serilog" Version="2.9.0" />
+    <PackageReference Include="Serilog.Sinks.Console" Version="3.1.1" />
+    <PackageReference Include="Serilog.Sinks.Trace" Version="2.1.0" />
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Utils\" />
```

Comment:
What is this for?

Reply:
It's a folder...  where the `Logger` class lives.  I don't guess I understand the question...

Reply:
Sorry if I was unclear: why do we need this? We don't need it for other directories (e.g, `Metrics`, `Configuration`)

Reply:
Oh, this is probably what you get when you add the folder via VS instead of VSCode...

Reply:
Removed it and nothing broke. :)